src/foam/nanos/crunch/Capability.js

```diff
@@ -197,6 +197,13 @@ foam.CLASS({
       javaFactory: 'return foam.mlang.MLang.TRUE;',
       documentation: 'Predicate used to omit or include capabilities from capabilityDAO'
     },
+    {
+      class: 'foam.mlang.predicate.PredicateProperty',
+      name: 'capabilityVisibilityPredicate',
```

Comment:
```suggestion
      name: 'visibilityPredicate',
```
@yij793 I think `visibilityPredicate` is better because it's consistent with `availabilityPredicate` and more generic

Reply:
yes, my initial comment on this was from misunderstanding.  So it should revert to visibilityPredicate.  Sorry about that.